### PR TITLE
[PDE-3149] docs(ui): Update S002 Integration Check

### PR DESCRIPTION
**Pull request by RasHanz**
Add further details to the S002 validation check about free trials/extensions for testing more than 5 triggers/actions/searches.

### CODE REVIEW

**Comment by RasHanz** [comment]
I am providing a bit more context for this one.
This change is based on an Area of opportunity from the [Escalations Ticket Taxonomy Recap - Q1 2022](https://async.zapier.com/p/61156/escalations-ticket-taxonomy-recap-q1-2022). Specifically:

> There tends to be confusion for new developers who have more than 5 Triggers and Actions that are interesting in submitting an app for review, but run into the #S002 validation check. - They can typically be blocked by not having a paid Zapier account and not being able to turn live Zaps on to pass this check. We have an opportunity here to add further details inviting them to reach out for a paid trial extension or engage in a free trial.

Added @owarij as a reviewer to check if this covers it.
I'm also adding @szchenghuang explicitly as a code owner.

Please let me know if I should wait for both approvals to merge. Otherwise, I'll proceed when one has approved.

Thanks!

**Review by xavdid — APPROVED** [review]
Looks good! Same thing as the other PR though- please also make this change in the monolith.

**Pull request merged**
